crates/next-core/js/src/ipc/index.ts

```diff
@@ -0,0 +1,144 @@
+import { createConnection } from "node:net";
```

Comment:
The duplication between `next-core` and `turbopack-node` is really confusing. Do we need both?

Reply:
Currently both are used, but that's not great. Eventually next-core should use turbopack-node for the SSR stuff too. But let's move that to a later PR